Main.java

```diff
@@ -0,0 +1,14 @@
+public class Main
+{
+    public static void main(String[] args){
+    if(args.length>0){
+        for(int i=0;i<args.length;i++){
+               System.out.println(args[i]);
+        }
+    }
+    else{
+        System.out.println("No argument given.");
+    }
+    }
+
+}
```

Comment:
Add a newline to the end of the file (https://stackoverflow.com/questions/4700312/empty-new-line-at-the-end-of-the-java-source-files).